docs/section6/Rule6-14.md

```diff
@@ -0,0 +1,41 @@
+
+# Lighting - Rule 6-14  
+
+**Rule ID:** 6-14  
+**Rule Description:** Proposed building is modeled with other programmable lighting controls through a 10% schedule reduction in buildings less than 5,000sq.ft.  
+**Rule Assertion:** Proposed RMR = expected value  
+**Appendix G Section:** Section G3.1-6(i) Modeling Requirements for the Proposed design  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:**  
+
+  1. Building total area is less than 5,000sq.ft.  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic:  
+
+- **Applicability Check 1:** ```if sum(space.floor_area for zone.space in P_RMR) < 5000:```  
+
+- For each building_segment in the Proposed model: ```For building_segment_p in P_RMR.building.building_segments:```  
+
+  - For each thermal_block in building segment: ```thermal_block_p in building_segment_p.thermal_blocks:```  
+
+    - For each zone in thermal block: ```zone_p in thermal_block_p.zones:```  
+
+      - For each space in thermal zone: ```space_p in zone_p.spaces:```  
+
+        - Get interior lighting in space: ```interior_lighting_p =  space_p.interior_lighting```  
+
+          - Get the EFLH value for the associated lighting schedule: ```schedule_EFLH_p = EFLH(interior_lighting_p.lighting_schedule_name)```  
```

Comment:
As far as I understand, we would need to do the following
1. Check if the zone has daylighting controls. If yes, check the flag for "•	are_schedules_used_for_modeling_daylighting_control" if TRUE, this would require a manual review.
2. Check if the zone has occupancy controls. For this, we would check the flag for are_schedules_used_for_modeling_occupancy_control. If TRUE, this would require a manual review.
3. If dayligthing controls or not present, or if are_schedules_used_for_modeling_daylighting_control is false and if are_schedules_used_for_modeling_occupancy_control is false , then we would do a comparison of each hourly schedule value between the B-RMR and P-RMR and verify if the value = 10%. 
4. If the value = 10%, then PASS
5. In all other cases, we have a CAUTION outcome with a requirement for manual review.

@mkarpman please weigh in

Reply:
@supriyagoel I think it should be like this:

1. If zone has daylighting_control_type!=none AND are_schedules_used_for_modeling_daylighting_control=TRUE, do manual review.
2. If zone occupancy_control_type!=none AND each hourly space lighting schedule fraction in P-RMR = 0.9*B-RMR then PASS.
3. If zone occupancy_control_type=none AND hourly schedule value in P-RMR = B-RMR then PASS.
4. In all other cases Fail